rust/otap-dataflow/crates/otap/src/otap_exporter.rs

```diff
@@ -116,17 +116,23 @@ impl local::Exporter<OtapPdata> for OTAPExporter {
             .await;
 
         let exporter_id = effect_handler.exporter_id();
-        let channel = Channel::from_shared(self.config.grpc_endpoint.clone())
-            .map_err(|e| {
+        let mut endpoint =
+            Channel::from_shared(self.config.grpc_endpoint.clone()).map_err(|e| {
                 let source_detail = format_error_sources(&e);
                 Error::ExporterError {
-                    exporter: exporter_id,
+                    exporter: exporter_id.clone(),
```

Comment:
nit: this clone is unnecessary isn't it?
```suggestion
                    exporter: exporter_id,
```